src/components/filters/AttributeFilter/AttributeFilter.tsx

```diff
@@ -13,8 +13,7 @@ import { setTelemetryHeaders } from "../../../helpers/utils";
 
 export interface IAttributeFilterProps {
     sdk?: SDK;
-    uri?: string;
-    identifier?: string;
+    filter: any; // JZA TODO: make the types more specific (FET-282)
```

Comment:
AFM.FilterItem?

Reply:
This is very important, it should be part of this ticket

Reply:
Done.